src/main/webapp/js/instructorFeedbackResults.js

```diff
@@ -209,14 +209,21 @@ function toggleCollapse(e, pans) {
     var panels = pans || $('div.panel-collapse');
 
     if ($(e).html().trim().startsWith(expand)) {
-        isExpandingAll = true;
-        var i = 0;
-        for (var idx = 0; idx < panels.length; idx++) {
-            if ($(panels[idx]).attr('class').indexOf('in') === -1) {
-                setTimeout(showSingleCollapse, 50 * i, panels[idx]);
-                i++;
+        $('.ajax-response-auto').click();
```

Comment:
Haha can I create another issue after my PR is merged. I will detailly explain what is going on here and why this is not desirable. I am not sure about how this could be improved. Should I just leave some comments in the code?